python/CMakeLists.txt

```diff
@@ -1,12 +1,20 @@
+CONFIGURE_FILE(${PY_NAME}/path.py.in ${PY_NAME}/path.py)
 SET(${PROJECT_NAME}_PYTHON_FILES
   robots_loader.py
   __main__.py
   __init__.py
   )
 
+IF(PROJECT_SITELIB_ROOT)
+  SET(${PY_NAME}_INSTALL_DIR ${PROJECT_SITELIB_ROOT}/${PYTHON_SITELIB}/${PY_NAME})
+ELSE()
+  SET(${PY_NAME}_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/${PYTHON_SITELIB}/${PY_NAME})
+ENDIF()
+
 FOREACH(python ${${PROJECT_NAME}_PYTHON_FILES})
-  PYTHON_INSTALL_ON_SITE(${PY_NAME} ${python})
+  INSTALL(FILES
```

Comment:
`PYTHON_INSTALL_ON_SITE` builds and install. This is useful to check the syntax and the build step, and to get a working python module in the build tree, which can be used by unit tests.